src/utils/perpareReciterSurahsList.js

```diff
@@ -13,7 +13,7 @@ export default (reciter, chapters, chapterId) => {
       id: chapters[index].id,
       path: `https://download.quranicaudio.com/quran/${reciter.relative_path}${zeroFill(3, chapters[index].id)}.mp3`,
       title: chapters[index].name.simple,
-      artist: reciter.name,
+      artist: reciter.name || 'hello',
```

Comment:
hello? maybe we can have a more applicable default reciter title here. Maybe "Reciter"